src/main/java/org/folio/edge/orders/OrdersHandler.java

```diff
@@ -103,7 +104,13 @@ protected void handleGobiResponse(RoutingContext ctx, HttpClientResponse resp) {
         }
 
         try {
-          String xml = parseResponse(resp, respBody).toXml();
+          String xml = StringUtils.EMPTY;
+          if(respBody.contains(VALIDATE_SUCCESS)){
```

Comment:
I don't understand why we need this...  If we keep this it needs to be formatted

Reply:
This handler method is used for both Post orders and the validate method. In case of validate we do not need to parse it to xml, but can send the received message from mod-gobi as-is.

Reply:
OK.  Given the expected usage it's probably an unnecessary optimization, but reasonable nonetheless.  I would however suggest that you replace `contains(...)` with `startsWith`, or better yet a regex that ignores case.  The current implementation could result in unexpected behavior... Consider the (admittedly contrived) case a JSON response is returned and some free-form notes field contains the text `<test>`.  Now we're returning JSON to GOBI (oops).  Another safeguard would be to still check the content-type header...   When you get down to it, after adding the necessary checks, it isn't much of an optimization, especially considering how simple the XML is.  Parsing it shouldn't be very expensive in the first place.  Just my 2 cents.

Reply:
The reason for not using the existing parser, is that the response fields are different for validate <test> vs <Response>. I tried not to over complicate the change by creating another wrapper class just for the <test>. However, I will change the contains to use a regex.

Reply:
Ah, right...  regex is prob a good compromise.

Reply:
Done!